docs/src/main/sphinx/release/release-402.md

```diff
@@ -0,0 +1,108 @@
+# Release 402 (2 Nov 2022)
+
+## General
+
+* Fix query hanging indefinitely when [fault-tolerant execution](/admin/fault-tolerant-execution)
+  is enabled and a [stage](trino-concept-stage) of the query produces no data. ({issue}`14794`)
+
+## Blackhole connector
+
+* Add support for column comments on view and materialized view columns. ({issue}`10705`)
+
+## Clickhouse connector
+
+* Reuse an existing JDBC connection for metadata queries. This can be disabled
+  with the `query.reuse-connection` configuration property. ({issue}`14653`)
+
+## Delta Lake connector
+
+* Remove the deprecated `hive.parquet.fail-on-corrupted-statistics` and
```

Comment:
Are they removed from the docs?

Reply:
Yeah, I think they were removed when they were deprecated.